packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -191,6 +203,7 @@ class RecoveryPhoneHandler {
               uid,
             }
           );
+          return { nationalFormat };
```

Comment:
Should we maybe return both the phoneNumber and the nationalFormat, might be a better API design. Also it's unlikely but possible that the phoneNumber exists, but the nationalFormat does not...